src/vs/base/browser/ui/sash/sash.ts

```diff
@@ -280,7 +286,6 @@ export class Sash extends Disposable {
 
 			dispose(disposables);
 
-			const iframes = getElementsByTagName('iframe');
```

Comment:
I removed this so that we use the same set of frames we selected previously (`iframes` was shadowed here). No need to select them twice--even if there's new iframes/webviews that appeared whilst dragging, we don't need to 'reset' their pointer events since we didn't touch them before.